### PR TITLE
Add 1 week as timespan option

### PR DESCRIPTION
**Pull request by Wolvan**
As requested in new-xkit/XKit#833, have a 1 week timespan

### CODE REVIEW

**Comment by izub** [comment]
Tested, @homu r+

File: Extensions/find_inactives.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
 //* TITLE Find Inactives **//
-//* VERSION 0.2.1 **//
+//* VERSION 0.3.0 **//
 //* DESCRIPTION Find the inactive blogs you follow **//
-//* DEVELOPER STUDIOXENIX **//
-//* DETAILS This extension lets you find the blog that haven't been updated for over 30 days. Just go to list of blogs you follow, then click on &quot;Find Inactive Blogs&quot; button below your Crushes to get started. **//
+//* DEVELOPER new-xkit **//
+//* DETAILS This extension lets you find the blog that haven't been updated for a certain amount of time. Just go to list of blogs you follow, then click on &quot;Find Inactive Blogs&quot; button below your Crushes to get started. **//
```
Comment (nightpool) [inline Extensions/find_inactives.js]: the existing description needs some grammatical help. maybe "The extension lets you find blogs that haven't been updated recently." for the first sentence? the second sentence looks fine.

**Comment by nightpool** [comment]
LGTM with updated description.

File: Extensions/find_inactives.js (hunk 1 above)
Comment (Wolvan) [inline Extensions/find_inactives.js]: I would not see 1 year as recently

Comment (nightpool) [inline Extensions/find_inactives.js]: fair enough. my main point was the grammar—"This extension lets you find blogs that haven't been updated in a certain amount of time" is certainly fine.

still LGTM with fixed description.

Comment (Wolvan) [inline Extensions/find_inactives.js]: ....Holy shit, I didn't even notice the horrible grammar in this sentence. Excuse that, @nightpool, I am definitely gonna fix that

**Comment by nightpool** [comment]
LGTM @homu r+

**Pull request merged**